content/en/docs/components/pipelines/v2/installation/quickstart.md

```diff
@@ -51,9 +38,58 @@ After you deploy Kubernetes, obtain your KFP endpoint by following [these instru
 </details>
 
 <details>
-<summary><a name="kfp_qs_step2"></a><h2 style="display:inline;">Step 2: Create and run a simple pipeline using the KFP SDK</h2></summary>
+<summary><a name="kfp_qs_ui"></a><h2 style="display:inline;">Run a basic pipeline on the KFP Dashboard</h2></summary>
+
+Kubeflow Pipelines offers a few samples that you can use to try out
+Kubeflow Pipelines quickly. The steps below show you how to run a basic sample that
+includes some Python operations, but doesn't include a machine learning (ML) 
+workload:
+
+1. Click the name of the sample, **[Tutorial] Data passing in python components**, on the pipelines UI:
+  <img src="/docs/images/v2/click-pipeline-sample.png" 
+    alt="Pipelines UI"
+    class="mt-3 mb-3 border border-info rounded">
+
+2. Click **Create experiment**:
+  <img src="/docs/images/v2/pipelines-start-experiment.png" 
+    alt="Creating an experiment on the pipelines UI"
+    class="mt-3 mb-3 border border-info rounded">
 
-This step shows how to use the KFP SDK to compose a pipeline and submit it for execution by KFP.
+3. Follow the prompts to create an **experiment** and then create a **run**. 
+  The sample supplies default values for all the parameters you need. The 
+  following screenshot assumes you've already created an experiment named
+  _My experiment_ and are now creating a run named _My first run_:
+  <img src="/docs/images/v2/pipelines-start-run.png" 
+    alt="Creating a run on the pipelines UI"
+    class="mt-3 mb-3 border border-info rounded">
+
+4. Click **Start** to run the pipeline.
+
+5. Explore the graph and other aspects of your run by clicking on the nodes 
+  (components) of the graph and the other UI elements:
+  <img src="/docs/images/v2/pipelines-basic-run.png" 
+    alt="Run results on the pipelines UI"
+    class="mt-3 mb-3 border border-info rounded">
+
+You can find the [source code for the **Data passing in python components** tutorial](https://github.com/kubeflow/pipelines/tree/sdk/release-1.8/samples/tutorials/Data%20passing%20in%20python%20components) in the Kubeflow Pipelines repo.
```

Comment:
We probably should use the v2 or above tutorial source code. v1.8 is already outdated.

https://github.com/kubeflow/pipelines/tree/2.0.0/samples/tutorials/Data%20passing%20in%20python%20components